src/lucky.cr

```diff
@@ -80,22 +84,31 @@ options = OptionParser.new do |parser|
         To compile your tasks, build the tasks.cr file
         > crystal build tasks.cr -o bin/tasks
 
+        Then run your tasks file with your task
+        > ./bin/tasks my.task -m --task=1
+
         MESSAGE
         exit
-      }
-    }
+      end
+    end
+
+    if ARGV.size > 1 && !built_in_commands.includes?(task_name)
+      # This stops the CLI parsing options beyond this point.
+      # It allows args to be passed to custom CLI tasks
+      parser.stop
+    end
```

Comment:
This stops the parser from parsing `-h` as the help for the `lucky` command. But we don't want to stop the parser when running internal commands like `lucky tasks -h` or `lucky dev -h`.